src/Fixer/Phpdoc/AlignMultilineCommentFixer.php

```diff
@@ -0,0 +1,155 @@
+with a line not prefixed with asterisk
+
+   */'
+                ),
+                new CodeSample(
+'<?php
+    /*
+            * This is a doc-like multiline comment
+*/',
+                    ['comment_type' => 'phpdocs_like']
+                ),
+                new CodeSample(
+'<?php
+    /*
+            * This is a doc-like multiline comment
+with a line not prefixed with asterisk
+
+   */',
+                    ['comment_type' => 'all_multiline']
+                ),
+            ]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isAnyTokenKindsFound($this->tokenKinds);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function applyFix(\SplFileInfo $file, Tokens $tokens)
+    {
+        $lineEnding = $this->whitespacesConfig->getLineEnding();
+        foreach ($tokens as $index => $token) {
+            if (!$token->isGivenKind($this->tokenKinds)) {
+                continue;
+            }
+
+            $whitespace = '';
+            $previousIndex = $index - 1;
+            if ($tokens[$previousIndex]->isWhitespace()) {
+                $whitespace = $tokens[$previousIndex]->getContent();
+                --$previousIndex;
+            }
+            if ($tokens[$previousIndex]->isGivenKind(T_OPEN_TAG)) {
+                $whitespace = preg_replace('/\S/', '', $tokens[$previousIndex]->getContent()).$whitespace;
+            }
+
+            if (1 !== preg_match('/\R([ \t]*)$/', $whitespace, $matches)) {
+                continue;
+            }
+
+            $indentation = $matches[1];
+            $lines = preg_split('/\R/', $token->getContent());
+            foreach ($lines as $lineNumber => $line) {
+                if (0 === $lineNumber) {
+                    continue;
+                }
+
+                $line = ltrim($line);
+                if ($token->isGivenKind(T_COMMENT) && (!isset($line[0]) || '*' !== $line[0])) {
+                    if ('all_multiline' !== $this->configuration['comment_type']) {
+                        continue 2;
+                    }
+                    continue;
+                }
+
+                if (!isset($line[0])) {
+                    $line = '*'.$line;
+                } elseif ('*' !== $line[0]) {
+                    $line = '* '.$line;
+                }
+
+                $lines[$lineNumber] = $indentation.' '.$line;
+            }
+
+            $tokens[$index] = new Token([$token->getId(), implode($lineEnding, $lines)]);
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function createConfigurationDefinition()
+    {
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('comment_type', 'Whether to align doc-like multiline comments if all lines start with an asterisk [`phpdocs_like`] or all multile comments  with mixed content on lines that start with an asteristk [`all_multiline`]'))
```

Comment:
I would update the description: _Whether to fix PHPDoc comments only (`phpdocs_only`), any multi-line comment whose lines all start with an asterisk (`phpdocs_like`) or any multi-line comment (`all_multiline`)._